pkg/auth/model/model.go

```diff
@@ -101,52 +101,35 @@ type BaseUser struct {
 	ExternalID        *string `db:"external_id" json:"external_id"`
 }
 
-type User struct {
-	ID string
-	BaseUser
-}
-
 type DBUser struct {
 	ID int64 `db:"id"`
 	BaseUser
```

Comment:
I suggest a rename just like with `Group` and `Policy`
```suggestion
	User
```